index.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 var isFinite = require('is-finite');
 
-module.exports = function (numbers) {
+module.exports = function (numbers, opts) {
 	if (!Array.isArray(numbers)) {
 		throw new TypeError('Expected an array');
 	}
 
 	var ticks = ['▁', '▂', '▃', '▄', '▅', '▆', '▇', '█'];
 	var finiteNumbers = numbers.filter(isFinite);
-	var min = Math.min.apply(null, finiteNumbers);
-	var max = Math.max.apply(null, finiteNumbers);
+  opts = opts || {};
```

Comment:
No this is correct. It will use the variable declared in the function declaration. It's a neat way to set a default value for an undefined parameter.